pgMasker_test.go

```diff
@@ -0,0 +1,56 @@
+package masker
+
+import (
+	"testing"
+)
+
+func TestMaskValidPgQuery(t *testing.T) {
```

Comment:
It is good to prepare tests for PgQuery 👍 
But extra package may change behavior from one we expect.
So, I propose that you introduce to gomodules.

Reply:
48b4f21
I introduce gomodules. Thank you for your proposal.